lib/components/programs-list/index.tsx

```diff
@@ -0,0 +1,104 @@
+import { Flex, Heading, Text, Box, Link, Button, useBreakpoint } from '@chakra-ui/react';
+import { PAGE_SIZES } from '../../constants';
+
+const programs: ProgramListingProps[] = [
+  {
+    title: 'LAUNCH PARTY',
+    description:
+      'Join OutstandingLife for a fun and informative conversation about how we can build an online community together.',
+    date: 'Friday, June 3',
```

Comment:
❓ Why is the date in this format?

Reply:
It's temporary just to match their programs design. We will change it to datetime when the backend is updated.